signalfx-java/src/main/java/com/signalfx/metrics/connection/AbstractHttpDataPointProtobufReceiverConnection.java

```diff
@@ -63,20 +69,31 @@ protected abstract HttpEntity getEntityForVersion(
             List<SignalFxProtocolBuffers.DataPoint> dataPoints);
 
     @Override
-    public void backfillDataPoints(String auth, String source, String metric,
-                                   List<SignalFxProtocolBuffers.Datum> datumPoints)
+    public void backfillDataPoints(String auth, String metric, String metricType, String orgId, Map<String,String> dimensions,
```

Comment:
Why is the `orgId` required is this API? Any particular reason why the backend doesn't derive it from the given auth token like we do for normal ingest?

Reply:
Great question! I'm not sure, I just followed [what the documentation told me](https://developers.signalfx.com/backfill_reference.html).